test/utils/CommonActions.js

```diff
@@ -145,48 +126,6 @@ class CommonActions {
         return this.getElement(elementCss).getText().toString() === textToVerify;
     }
 
-    /**
-     * @param elementCss auto completer textField.
-     * @param textToSelect select text on autoCompleter.
-     * @param autoCompleterKey locator to auto completer button on classic theme.
-     */
-    static selectOnAutoCompleterTextField(elementCss, textToSelect, autoCompleterKey) {
-        this.setInputTextField(elementCss, textToSelect);
-        if (isClassic) {
-            this.clickWebElement(autoCompleterKey);
-            this.autoCompleterClassicTheme(textToSelect);
-        }
-        else {
-            this.autoCompleterLightTheme(textToSelect, autoCompleterKey);
-        }
-
-    }
-
-
-    /**
-     * @param textToSelect select text on autoCompleter.
-     */
-    static autoCompleterClassicTheme(textToSelect) {
```

Comment:
Are you sure? this methods is to AutoCompleter Field on Classic and Lightning theme, maybe you need think this.

Reply:
the same as above